### DISCUSSION TITLE
2.x Late Bloomer

### DISCUSSION THREAD

**Discussion by Acewiza** [Security-Onion-Solutions/securityonion#2471] [General] [answered]
Took awhile to figure out how to get it installed the way I wanted on this hardware, but after a few days working with it, all I can say is WOW - light years ahead of 16.  It's easy to see all the hard work that went into this.  Big props to the SO team, all around!

**Comment by dougburks** [comment, marked as answer]
Thanks @Acewiza, glad you like it!
